### ISSUE TITLE
./configure not found in boehm-7.2

### ISSUE THREAD

**Issue by robert-strandh** [drmeister/legacy-externals-clasp#3]
When I follow the instructions and type `make' in order to make
externals-clasp, I can see that it does a`cd' into boehm-7.2 and then
it tries to execute ./configure, but there is no such file.  There is,
however, a file configure.ac in boehm-7.2.

**Comment by robert-strandh** [comment]
Part of the problem seems to have been that `automake' was not
installed.  It might be worthwhile to add this requirement to the
build instructions.

However, even after I installed automake and made a fresh clone, It
fails.  I do have a message about LIBTOOL being undefined and that the
way to define it is to add LT_INIT to configure.ac and run `aclocal'.
But LT_INIT is already defined in boehm-7.2/configure.ac as far as I
can tell.  So I tried to run aclocal in the boehm-7.2/ directory, but
I still have the same problem, i.e., ./configure does not exist in
boehm-7.2.
